scalatest/src/main/scala/org/scalatest/exceptions/StackDepthException.scala

```diff
@@ -184,13 +184,13 @@ abstract class StackDepthException(
    *
    * @return the detail message string of this <code>StackDepthException</code> instance (which may be <code>null</code>).
    */
-  override def getMessage: String = message.orNull
+  override def getMessage= message.orNull.asInstanceOf[String]
```

Comment:
**Overridden java method return type cannot be null**
The nullness in the return type of all java methods are cast away by the `-Yjava-interop-dont-nullify-outermost` compiler flag. `asInstanceOf[A]` is used for return values of type `A | Null`.